ops/model.py

```diff
@@ -195,7 +195,7 @@ def uuid(self) -> str:
     def get_unit(self, unit_name: str) -> 'Unit':
         """Get an arbitrary unit by name.
 
-        Use :attr:`unit` to get your own unit.
+        Use :attr:`unit` to get the model's own unit.
```

Comment:
Not sure about this, as "the model" has several units. Perhaps "the current unit"?